src/noisepy/seis/S2_stacking.py

```diff
@@ -193,7 +187,7 @@ def stack(cc_store: CrossCorrelationDataStore, stack_dir: str, fft_params: Confi
             # jump if there are not enough data
             if len(indx) < 2:
                 iflag = 0
-                break
+                continue
```

Comment:
@mdenolle - While testing the scedc notebook I was getting errors when trying to plot the `ZZ` stack. I found this component was not being computed for some pairs because one of the previous components (`NE` I think) didn't have enough data. Seems like this should be `continue` and not a `break`?